specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -1564,922 +2662,1213 @@
+          "description": "GeoJSON object. Displayed when radiiGeometry=true in request. Describes the outline of the wind radius quadrants."
         }
       }
     },
-    "HazardDetail": {
+    "RadiusSector": {
       "type": "object",
+      "description": "Bearing 0–90 = NE quadrant; 90–180 = SE quadrant; 180–270 = SW quadrant; 270–360 = NW quadrant.",
       "properties": {
-        "hazardIndex": {
-          "$ref": "#/definitions/HazardIndex"
+        "beginBearing": {
+          "type": "number",
+          "format": "double",
+          "description": "Bearing, in degrees, of the beginning of the quadrant."
         },
-        "hazardCode": {
-          "type": "string",
-          "description": "A unique identifier (non-displayable) for each type of hazard: LightRain, ModerateRain, HeavyRain, LightMix, ModerateMix, HeavyMix, LightSnow, ModerateSnow, HeavySnow, LightIce, ModerateIce, HeavyIce, Hail, LargeHail, SunGlare, SunGlareHigh, Lightning, SevereLightning, WindModerate, WindHigh, WindExtreme, FloodWarning, FlashFloodWarning, TornadoWarning, TsunamiWarning, SevereThunderstormWarning."
+        "endBearing": {
+          "type": "number",
+          "format": "double",
+          "description": "Bearing, in degrees, of the end of the quadrant."
         },
-        "shortPhrase": {
-          "x-ms-client-name": "shortDescription",
-          "type": "string",
-          "description": "A displayable short phrase describing the forecasted conditions and precipitation intensity/type."
+        "range": {
+          "type": "number",
+          "format": "double",
+          "x-ms-client-name": "radius",
+          "description": "The radius of the quadrant, in nautical miles."
         }
       }
     },
-    "WeatherNotification": {
+    "StormLocationsResult": {
       "type": "object",
+      "description": "Locations for an individual government-issued storm",
       "properties": {
-        "type": {
+        "results": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LocationResult"
+          },
+          "description": "Locations for an individual government-issued storm"
+        }
+      }
+    },
+    "LocationResult": {
+      "type": "object",
+      "description": "Location for an individual Government-issued storm",
+      "properties": {
+        "dateTime": {
           "type": "string",
-          "description": "A type of notification generated to warn drivers of the onset of a hazard, or increase in intensity of a hazard."
+          "x-ms-client-name": "timestamp",
+          "description": "Datetime the forecast is valid, displayed in ISO8601 format."
         },
-        "hazardIndex": {
-          "$ref": "#/definitions/HazardIndex"
+        "location": {
+          "$ref": "../../../Common/preview/1.0/common.json#/definitions/LatLongPair",
+          "x-ms-client-name": "coordinate",
```

Comment:
I think the consistent usage in maps APIs is "coordinates"

Reply:
Updated